src/Bicep.LangServer.UnitTests/BicepCompletionProviderTests.cs

```diff
@@ -368,17 +368,78 @@ public void ParameterTypeContextShouldReturnDeclarationTypeCompletions()
                     c.Kind.Should().Be(CompletionItemKind.Snippet);
                     c.InsertTextFormat.Should().Be(InsertTextFormat.Snippet);
                     c.TextEdit!.NewText.Should().StartWith("object");
-                    c.TextEdit.NewText.Should().Contain("secure: true");
+                    c.TextEdit!.NewText.Should().Be("object");
                     c.Detail.Should().Be("Secure object");
+                    c.AdditionalTextEdits?.Count().Should().Be(1);
```

Comment:
>AdditionalTextEdits [](start = 22, length = 19)

nit - The compiler carries the non-nullness forward once you use `!`, so it's enough to replace the `?` with `!` and don't need the following `!`s. (Same for all the changes below.)

Reply:
Hmmm it doesn't work consistently. Looks like a bug on roslyn side.

![image](https://user-images.githubusercontent.com/30270536/112743805-4ce20000-8f4f-11eb-98e6-ef995aec573a.png)

Reply:
Marking this resolved. Let me know if I am missing something. I can fix it up in a follow up PR. Thanks!